apps/codeforafrica/src/components/StayInTouch/Title.js

```diff
@@ -0,0 +1,29 @@
+import { RichTypography } from "@commons-ui/core";
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+const TypographyRoot = styled(RichTypography)(
+  ({ theme: { typography, breakpoints } }) => ({
+    textTransform: "uppercase",
+    fontWeight: "bold",
+    marginBottom: typography.pxToRem(23),
+    [breakpoints.up("md")]: {
+      marginBottom: 0,
+    },
+  })
+);
+
+function Title({ title }) {
+  return <TypographyRoot>{title}</TypographyRoot>;
+}
+
+Title.propTypes = {
+  title: PropTypes.string,
```

Comment:
```suggestion
  children: PropTypes.string,
```
I think `<Title>Goes here</Title>` is better than `<Title tile="Goes here" />`. I also think `<Typography variant="footerCap">` captures most of the styles.